advanced_functionality/scikit_bring_your_own/scikit_bring_your_own.ipynb

```diff
@@ -415,7 +418,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Deploy the model\n",
+    "## Hosting your model\n",
+    "One can use trained model to get real time predictions using HTTP endpoint. Following steps walk you through the process."
```

Comment:
"You can use a trained model to get real time predictions using HTTP endpoint. Follow these steps to walk you through the process."